### PR TITLE
Rewrite CSS best practice introduction paragraph

### PR DESCRIPTION
**Pull request by danfarrow**
I saw this paragraph in another PR & found the wording clumsy so I rewrote it

### CODE REVIEW

File: wordpress-coding-standards/css.md [hunk 1]
```diff
@@ -275,7 +275,7 @@ For inline:
 ```
 
 <h2>Best Practices</h2>
-Stylesheets tend to get long in length. Focus slowly gets lost whilst intended goals start repeating and overlapping. Writing smart code from the outset helps us retain the overview whilst remaining flexible throughout change.
+Stylesheets tend to grow in length &amp; complexity, and as they grow the chance of redundancy increases. By following some simple best practices we can help our CSS maintain focus and flexibility as it evolves:
```
Comment (jrfnl) [inline wordpress-coding-standards/css.md:278]:
```suggestion
Stylesheets tend to grow in length &amp; complexity, and as they grow the chance of redundancy increases. By following some best practices we can help our CSS maintain focus and flexibility as it evolves:
```

Ablist language: what may be "simple" for an experienced developer, may be a complex for someone new to the field.

Comment (danfarrow) [reply wordpress-coding-standards/css.md]: Agreed. I intended “simple” in the sense of “basic”, but “basic” also carries subjectivity so better to remove it altogether!

Comment (jrfnl) [reply wordpress-coding-standards/css.md]: If it helps: I often fall back to "straight forward" for these kind of phrases.

**Review by dingo-d — APPROVED** [review]

File: wordpress-coding-standards/css.md [hunk 2]
```diff
@@ -275,7 +275,7 @@ For inline:
 ```
 
 <h2>Best Practices</h2>
-Stylesheets tend to get long in length. Focus slowly gets lost whilst intended goals start repeating and overlapping. Writing smart code from the outset helps us retain the overview whilst remaining flexible throughout change.
+Stylesheets tend to grow in length &amp; complexity, and as they grow the chance of redundancy increases. By following some best practices we can help our CSS maintain focus and flexibility as it evolves:
```
Comment (GaryJones) [inline wordpress-coding-standards/css.md:278]:
```suggestion
Stylesheets tend to grow in length and complexity, and as they grow the chance of redundancy increases. By following some best practices we can help our CSS maintain focus and flexibility as it evolves:
```

No need for an ampersand here when an "and" works fine.

**Review by GaryJones — APPROVED** [review]

**Pull request merged**